cmd/meroxa/root/account/accounts.go

```diff
@@ -22,6 +22,8 @@ import (
 	"github.com/meroxa/cli/cmd/meroxa/builder"
 )
 
+// Account implement the projects management onto the Meroxa platform.
+// Accounts were renames Projects, renamed to fix the information architecture.
```

Comment:
@nassor Thanks for the context. I think this reads a bit clearer / more concise with the following change. What do you think? 
```suggestion
// Accounts on the platform were renamed to Projects, to fix the information architecture.
```